tests/unit/resource/resource_tests.cpp

```diff
@@ -42,6 +42,8 @@ TEST(DefaultMemoryResource, AllocateDeallocate)
   auto alloc = std::make_shared<umpire::resource::DefaultMemoryResource<TestAllocator> >(umpire::Platform::cpu, "TEST", 0, umpire::MemoryResourceTraits{});
```

Comment:
Totally orthogonal to this, but we don't need the shared pointer here.

Reply:
💯